Dockerfile

```diff
@@ -21,53 +19,31 @@ COPY --from=frontend-workspace /app/.moon/docker/sources .
 RUN moon run frontend:build transactional:build
 RUN moon docker prune
 
-FROM --platform=$BUILDPLATFORM lukemathwalker/cargo-chef AS backend-chef
-RUN apt-get update && apt-get install -y --no-install-recommends gcc-aarch64-linux-gnu libc6-dev-arm64-cross clang llvm ca-certificates pkg-config make g++ libssl-dev
-RUN update-ca-certificates
-WORKDIR app
-
-FROM backend-chef AS backend-planner
-COPY . .
-RUN cargo chef prepare --recipe-path recipe.json
-
-FROM backend-chef AS backend-builder
-# build specific
-ARG BUILD_PROFILE=release
-# application specific
-ARG APP_VERSION
-ARG DEFAULT_TMDB_ACCESS_TOKEN
-ARG DEFAULT_MAL_CLIENT_ID
-ARG DEFAULT_GOOGLE_BOOKS_API_KEY
-RUN test -n "$APP_VERSION" && \
-    test -n "$DEFAULT_TMDB_ACCESS_TOKEN" && \
-    test -n "$DEFAULT_MAL_CLIENT_ID" && \
-    test -n "$DEFAULT_GOOGLE_BOOKS_API_KEY"
-COPY --from=backend-planner /app/recipe.json recipe.json
-RUN cargo chef cook --profile $BUILD_PROFILE --recipe-path recipe.json
-COPY . .
-COPY --from=frontend-builder /app/apps/backend/templates ./apps/backend/templates
-RUN APP_VERSION=$APP_VERSION \
-    DEFAULT_TMDB_ACCESS_TOKEN=$DEFAULT_TMDB_ACCESS_TOKEN \
-    DEFAULT_MAL_CLIENT_ID=$DEFAULT_MAL_CLIENT_ID \
-    DEFAULT_GOOGLE_BOOKS_API_KEY=$DEFAULT_GOOGLE_BOOKS_API_KEY \
-    cargo build --profile ${BUILD_PROFILE} --bin ryot
-RUN cp -R /app/target/${BUILD_PROFILE}/ryot /app/ryot
+FROM --platform=${BUILDPLATFORM} alpine as artifact
+COPY artifact/ /artifact/
+ARG TARGETARCH
+ENV TARGETARCH=${TARGETARCH}
+RUN mv /artifact/backend-${TARGETARCH}/ryot /artifact/ryot
+RUN chmod +x /artifact/ryot
 
 FROM $NODE_BASE_IMAGE
+ARG TARGETARCH
+ENV TARGETARCH=${TARGETARCH}
 LABEL org.opencontainers.image.source="https://github.com/IgnisDa/ryot"
 ENV FRONTEND_UMAMI_SCRIPT_URL="https://umami.diptesh.me/script.js"
 ENV FRONTEND_UMAMI_WEBSITE_ID="5ecd6915-d542-4fda-aa5f-70f09f04e2e0"
 COPY --from=caddy:2.7.5 /usr/bin/caddy /usr/local/bin/caddy
-RUN apt-get update && apt-get install -y --no-install-recommends curl ca-certificates procps && rm -rf /var/lib/apt/lists/*
+RUN apt-get update && apt-get install -y --no-install-recommends libssl3 ca-certificates procps && rm -rf /var/lib/apt/lists/*
 RUN npm install --global concurrently@8.2.2 && concurrently --version
 RUN useradd -m -u 1001 ryot
+RUN if [ "${TARGETARCH}" = "arm64" ]; then apt-get update && apt-get install -y --no-install-recommends wget && wget http://ftp.debian.org/debian/pool/main/o/openssl/libssl1.1_1.1.1w-0+deb11u1_arm64.deb && dpkg -i libssl1.1_1.1.1w-0+deb11u1_arm64.deb && rm -rf libssl1.1_1.1.1w-0+deb11u1_arm64.deb && apt-get remove -y wget && rm -rf  rm -rf /var/lib/apt/lists/*; fi
```

Comment:
@IgnisDa the only manual installation is this line and it is only a workaround until `cross-rs` update their base image. Right now, we have
- `amd64` compiles on github runner which has `libssl3`.
- `arm64` compiles on `cross-rs` base image which has `libssl1.1`.

Finding a base image that have both of their libraries is impossible so i think we should just stick with this.

Reply:
Cool thanks